python/F3DPythonBindings.cxx

```diff
@@ -334,7 +334,33 @@ PYBIND11_MODULE(pyf3d, module)
     .def_static("load_plugin", &f3d::engine::loadPlugin, "Load a plugin")
     .def_static(
       "autoload_plugins", &f3d::engine::autoloadPlugins, "Automatically load internal plugins")
-    .def_static("get_plugins_list", &f3d::engine::getPluginsList);
+    .def_static("get_plugins_list", &f3d::engine::getPluginsList)
+    .def_static("get_lib_info", &f3d::engine::getLibInfo, py::return_value_policy::reference)
+    .def_static("get_readers_info", &f3d::engine::getReadersInfo);
+
+  // libInformation
+  py::class_<f3d::engine::libInformation>(module, "LibInformation")
+    .def(py::init<>())
```

Comment:
```suggestion
```
Should not be needed (same for `ReaderInformation`) unless we need to create this instances from python code for some reason?

Reply:
Ok